cmd/builder/cli/cli.go

```diff
@@ -62,6 +61,7 @@ func Execute() error {
 
 func init() {
 	builderCmd.Flags().StringSliceVarP(&buildpackList, "buildpack", "b", nil, "buildpack(s) to use")
+	builderCmd.Flags().StringVarP(&systemBuildpacksDir, "system-buildpacks-dir", "", "/tmp/buildpacks", "system buildpacks dir")
```

Comment:
Why not `os.TempDir()`?

Reply:
`system-buildpacks-dir` should point to the directory created by Diego, when it mounts system-buildpacks. This folder is read-only.

Reply:
Got it.